sycl/doc/SYCLCompilerUserManual.md

```diff
@@ -112,7 +112,15 @@ your application.
 
 **`-fintelfpga`**
 
-    Perform ahead of time compilation for FPGA.
+    Perform ahead of time compilation for Intel FPGA, which relies on the external tool `aoc`
+    being available in the `PATH`.
+    
+    This option is roughly equivalent to `-fsycl-targets=spir64_fpga-unknown-unknown-sycldevice -MMD -lOpenCL`.
```

Comment:
Should you also mention that it sets some flags that are usually desired by FPGA users?  E.g. -g.

Reply:
Thanks - I didn't notice the effect of `-g` because I am already using it, I've added it.